libs/vkd3d/command.c

```diff
@@ -2763,13 +2781,17 @@ static HRESULT d3d12_command_allocator_init(struct d3d12_command_allocator *allo
 
     allocator->current_command_list = NULL;
 
+    allocator->vk_interop = vk_interop;
```

Comment:
The thought here was to specifically note whether or not a command allocator was going to be used for interop. So that the interop functions wouldn't be used with non-interop enabled allocators. This extra validation step isn't strictly necessary, so I'm happy to get rid of it if you prefer.

Reply:
On further thought (and reading your comments below) this is unnecessary. I think a model described below (with command list state invalidation) makes more sense, and is a bit more forward-thinking.